### DISCUSSION TITLE
How do I create a comment on a pull request?

### DISCUSSION THREAD

**Discussion by aldairandre** [octokit/octokit.js#2413] [Q&A] [answered]
I was reading the rest API documentation but I didn't find what I need, can someone help me?

**Comment by wolfy1339** [comment, marked as answer]
GitHub treats pull requests and issues the same for comments.

You can simply use the Issue Comment endpoint with the pull request number.

https://docs.github.com/en/rest/issues/comments

**Comment by aldairandre** [comment]
Thanks for your help, man.  💪😁
